### ISSUE TITLE
import multiple local sql files

### ISSUE THREAD

**Issue by fisherwei** [dbeaver/cloudbeaver#1307]
SQL editor can open only one sql files, if we can open multiple files as same time will be better.

**Comment by kseniaguzeeva** [comment]
Could you please add more details in the ticket what feature do you want to see in CloudBeaver? How should it work?
You can open multiple SQL Editors for different connections for now by pressing the SQL button or via the context menu in the Navigator tre.

**Comment by fisherwei** [comment]
> Could you pleass add more details in the ticket what feature do you want to see in CloudBeaver? How should it work? You can open multiple SQL Editors for different connections for now by pressing the SQL button or via the context menu in the Navigator tre.

yes, we can open multiple tabs, but if I want to run many many .sql scripts, I have to one by one.
Adminer, it can upload multiple files at the same time, and then run them.

![iShot_2022-11-25_15 35 03](https://user-images.githubusercontent.com/214470/203925975-9ecd099b-018b-4158-8bf7-292c377946e5.png)

**Comment by miltonchirinos44** [comment]
@fisherwei you should change the title and as a suggestion it should be called **import multiple local sql files**

**Comment by kseniaguzeeva** [comment]
Thank you for the explanation. CloudBeaver runs in a browser, so the size of a loaded script has a limit. So, we should find a good solution to avoid freezing the application.
I'l inform you as soon as the way of the implementation is found.

**Comment by alexisdondon** [comment]
Could it be better to have a git integration to let git clone and git push script in workspace instead of upload/download scripts?

**Comment by devnaumov** [comment]
What is the anticipated action when multiple files are imported concurrently? Are they expected to appear sequentially, one after the other?

**Comment by LonwoLonwo** [comment]
Hello @alexisdondon 
We have GIT integration in our TE product: https://github.com/dbeaver/cloudbeaver/wiki/Team-Edition-Git
